apps/retrospective/services/sprint_task.go

```diff
@@ -74,7 +88,19 @@ func (service SprintTaskService) Get(
 	sprintTaskFilter := db.Model(&retroModels.SprintTask{}).Where("id = ?", sprintTaskID).
 		Select("task_id").QueryExpr()
 
-	dbs := service.tasksForCurrentSprint(retroID, sprintID).
+	taskOwnerTable := service.tasksForCurrentAndPrevSprint(retroID, sprintID).
+		Where("sprint_tasks.task_id = (?)", sprintTaskFilter).
+		Select(`
+        DISTINCT ON (tasks.id) tasks.id AS task_id,
+        users.first_name || ' ' || users.last_name          AS member_name,
+        SUM(sprint_member_tasks.time_spent_minutes)
+        OVER (
+          PARTITION BY tasks.id, sprint_members.member_id ) AS member_time`).
+		Order("tasks.id").
+		Order("member_time DESC").
+		QueryExpr()
```

Comment:
We can extract this into a method as most of it is common with the query used in the method above.

Reply:
done